nautobot/docs/installation/http-server.md

```diff
@@ -160,14 +160,40 @@ Begin by installing Apache:
 $ sudo apt install -y apache2
 ```
 
-Next, copy the default configuration file to `/etc/apache2/sites-available/`. Be sure to modify the `ServerName` parameter appropriately.
+Next, copy and paste the following Apache configuration file to `/etc/apache2/sites-available/nautobot.conf`. Be sure to modify the `ServerName` parameter appropriately:
 
-If the location of SSL certificates had to be changed in `Obtain an SSL Certificate` step, then the location will need to be changed in the `apache.conf` file as well.
+```apache
```

Comment:
I *think* you will need to add `apache` to the `hljs_languages:` list in `mkdocs.yml` for this to get highlighted appropriately, though I'd love to be wrong on this point.

Reply:
It would also be great to add `nginx` highlighting to the NGINX config earlier in this file, and enable it in the `mkdocs.yml` as well.

Reply:
Tried it and it didn't seem to do anything. :shrug: